examples/text_labels_and_annotations/text_rotation.py

```diff
@@ -0,0 +1,49 @@
+"""
+===========================
+Text rotation demonstration
+===========================
+
+The way matplotlib does text layout is counter-intuitive to some, so
```

Comment:
Maybe we could be a little gentler about this. Something like, "matplotlib takes a different approach to text layout..." (maybe?)